File: TripKitAndroid/src/main/java/com/skedgo/tripkit/Tables.java
package com.skedgo.tripkit;


import com.skedgo.sqlite.DatabaseField;
import com.skedgo.sqlite.DatabaseTable;

final class Tables {
  public static final DatabaseField FIELD_JSON = new DatabaseField("json", "TEXT");
  public static final DatabaseTable TRANSPORT_MODES = new DatabaseTable(
      "transport_modes",
      new DatabaseField[] {FIELD_JSON}
  );
  public static final DatabaseTable REGIONS = new DatabaseTable(
      "regions",
      new DatabaseField[] {FIELD_JSON}
  );

  private Tables() {}
}

File: TripKitAndroid/src/main/java/com/skedgo/tripkit/Tables.kt
package com.skedgo.tripkit

import com.skedgo.sqlite.DatabaseField
import com.skedgo.sqlite.DatabaseTable


internal object Tables {
    val FIELD_JSON: DatabaseField = DatabaseField("json", "TEXT")
    @JvmField
    val TRANSPORT_MODES: DatabaseTable = DatabaseTable(
        "transport_modes",
        arrayOf(FIELD_JSON)
    )
    @JvmField
    val REGIONS: DatabaseTable = DatabaseTable(
        "regions",
        arrayOf(FIELD_JSON)
    )
}